ccm_web/client/src/components/BulkEnrollUMUserToSectionsConfirmationTable.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+
+import CustomTable from './CustomTable'
+import { RowNumberedAddEnrollmentWithSectionId } from '../models/enrollment'
+
+interface BulkEnrollUMUserSectionsConfirmationTableProps {
+  enrollments: RowNumberedAddEnrollmentWithSectionId[]
+}
+
+interface TableHeaderColumnInfoShouldUseMatUIType {
+  id: keyof RowNumberedAddEnrollmentWithSectionId
+  label: string
+  minWidth: number
+  align?: 'left' | 'right' | undefined
+}
+
+const columns: TableHeaderColumnInfoShouldUseMatUIType[] = [
+  { id: 'rowNumber', label: 'Row Number', minWidth: 25 },
+  { id: 'loginId', label: 'Login ID', minWidth: 100 },
+  { id: 'role', label: 'Role', minWidth: 100 },
+  { id: 'sectionId', label: 'Section ID', minWidth: 100 }
```

Comment:
I like the flexibility the code is adding when uploading a csv file for single section, it is honoring multiple section csv file  as well. this is better if user does single and multiple often then he can deal with one file :)